src/parking-locations/types.ts

```diff
@@ -43,4 +48,31 @@ export type CleaningTime = {
   appliesToEvenWeeks: boolean,
   appliesToOddWeeks: boolean,
   noCleaningMonths: MonthOfYear[]
+}
+
+/* export class ReminderDataForBackend {
+  constructor(
+    public email: string,
+    public nextCleaningTime: DateTime | null,
+    public notificationDate: DateTime | null,
+    public carNickname: string,
+    public parkingSpotName: string,
+  ) { }
+} */
+
+export class UserInput {
+  constructor(
+    public email: string,
+    public carNickname: string,
+    public acceptTerms: boolean,
+    public notificationDate: DateTime | null,
+  ) { }
+}
+
+export class NotifUnsocialHours {
```

Comment:
NIT: When reading code, constructors that end up looking like `NotifUnsocialHours(true, true, true)` can be quite confusing, especially if not reviewing by checking out the code locally. One way of dealing with this is to pass in an object with labelled properties, especially if you end up with many optional values.

It is fine to leave it like this, but just a general tip!

Reply:
Great tip! I've found myself with similar looking constructors before so this is really applicable! I will sort this.

Reply:
Sorted and really liked this approach, thanks.